client/components/site-selector/add-site.tsx

```diff
@@ -10,13 +10,17 @@ const SiteSelectorAddSite: FunctionComponent = () => {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 
-	const recordAddNewSite = useCallback( () => {
-		const event = isJetpackCloud()
-			? 'calypso_add_new_jetpack_click'
-			: 'calypso_add_new_wordpress_click';
+	const recordAddNewSite = useCallback(
+		( e ) => {
```

Comment:
Let's add the type to this `event` argument here. I think it's `React.ClickEvent` or something like that.